static/app/components/dropdownAutoComplete/menu.tsx

```diff
@@ -491,14 +475,8 @@ const StyledDropdownBubble = styled(DropdownBubble)<{minWidth: number}>`
 const DropdownMainContent = styled('div')<{minWidth: number}>`
   width: 100%;
   min-width: ${p => p.minWidth}px;
-`;
-
-const InputWrapper = styled('div')`
-  display: flex;
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
-  border-radius: ${p =>
-    `calc(${p.theme.borderRadius} - 1px) calc(${p.theme.borderRadius} - 1px) 0 0`};
-  align-items: center;
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
```

Comment:
the border was on the wrong element, you can see that all children overlay the border otherwise